crates/hyperqueue/src/common/cli.rs

```diff
@@ -303,6 +308,25 @@ pub struct HwDetectOpts {
     pub no_hyper_threading: bool,
 }
 
+#[derive(Parser, Debug)]
+pub struct DeploySSHOpts {
+    /// Show log output of the spawned worker(s).
+    #[arg(long)]
+    pub show_output: bool,
+
+    /// Path to a nodefile that contains the hostnames to which should
+    /// HQ workers be deployed to.
+    /// Each line in the file should correspond to one hostname address.
```

Comment:
Can I add a hostname with port in case that SSH is not running on a standard port? (e.g. "myserver:1234")

Reply:
Good idea, added support for that.